apps/ledger-live-mobile/src/screens/SyncOnboarding/SoftwareChecksStep.tsx

```diff
@@ -190,15 +187,11 @@ const SoftwareChecksStep = ({ device, isDisplayed, onComplete }: Props) => {
 
         // Updates the genuineCheckStatus
         if (genuineCheckError) {
-          console.error(
-            "Failed to perform genuine check with error:",
-            genuineCheckError.message,
-          );
           setGenuineCheckStatus("failed");
         } else if (genuineState === "genuine") {
           setGenuineCheckStatus("completed");
         } else if (genuineState === "non-genuine") {
-          // FIXME: if the device is non-genuine, we should display something else
+          nextDrawerToDisplay = "cancelled";
           setGenuineCheckStatus("failed");
```

Comment:
You can remove the case `"failed"` now for the `GenuineCheckStatus`, it is not needed as now we have the `"skipped"` value.
And so you can remove the `else if` check where i've put a 🧹 in the PR comment